### PR TITLE
1.3.2 (UI Changes)

### PR DESCRIPTION
**Pull request by MrMendelli**
Major Changes:
* Updated placeholders.
* Changed main form size _(more ideal for previewing)_. Starts centered now.

Minor Changes:
* Changed program icon to Aurora logo.
* Updated and changed various UI strings for consistency.

Misc.:
* Reorganized source files.
* Updated readme.
* Updated `AssemblyInfo.cs` accordingly for releases.

### CODE REVIEW

File: AuroraAssetEditor/Classes/XboxAssetDownloader.cs [hunk 1]
```diff
@@ -177,12 +177,12 @@ public enum XboxAssetType {
                 }
             }
             titleInfo.AssetsInfo = ret.ToArray();
-            XboxAssetDownloader.SendStatusChanged("Finished parsing Title/Asset info...");
+            XboxAssetDownloader.SendStatusChanged("Finished fetching assets!");
```
Comment (Swizzy) [inline AuroraAssetEditor/Classes/XboxAssetDownloader.cs:180]: This should probably be "Finished parsing asset information..."

File: AuroraAssetEditor/Classes/XboxAssetDownloader.cs [hunk 2]
```diff
@@ -267,7 +267,7 @@ private static void ParseHtml(string url, XboxTitleInfo titleInfo)
             catch {  }
 
             titleInfo.AssetsInfo = ret.ToArray();
-            XboxAssetDownloader.SendStatusChanged("Finished parsing Title/Asset info...");
+            XboxAssetDownloader.SendStatusChanged("Finished fetching assets!");
```
Comment (Swizzy) [inline AuroraAssetEditor/Classes/XboxAssetDownloader.cs:270]: This should probably be "Finished fetching online assets..."

File: AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs [hunk 3]
```diff
@@ -79,7 +79,7 @@ public partial class OnlineAssetsControl {
             _unityWorker.DoWork += (o, args) => {
                                        try {
                                            _unityResult = XboxUnity.GetUnityCoverInfo(args.Argument.ToString());
-                                           Dispatcher.Invoke(new Action(() => StatusMessage.Text = "Finished downloading asset information..."));
+                                           Dispatcher.Invoke(new Action(() => StatusMessage.Text = "Finished fetching assets!"));
```
Comment (Swizzy) [inline AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs:82]: This should probably be "Finished fetching asset information..."

File: AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs [hunk 4]
```diff
@@ -108,12 +108,12 @@ public partial class OnlineAssetsControl {
                                           _xboxResult = _keywords == null
                                                             ? _xboxAssetDownloader.GetTitleInfo(_titleId, args.Argument as XboxLocale)
                                                             : _xboxAssetDownloader.GetTitleInfo(_keywords, args.Argument as XboxLocale);
-                                          Dispatcher.Invoke(new Action(() => StatusMessage.Text = "Finished downloading asset information..."));
+                                          Dispatcher.Invoke(new Action(() => StatusMessage.Text = "Finished fetching assets!"));
```
Comment (Swizzy) [inline AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs:111]: This should probably be "Finished fetching asset information..."

File: AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs [hunk 5]
```diff
@@ -290,7 +290,7 @@ public partial class OnlineAssetsControl {
                                          asset.GetCover();
                                  };
                     bw.RunWorkerCompleted += (o, args) => {
-                                                 StatusMessage.Text = "Finished downloading asset data...";
+                                                 StatusMessage.Text = "Finished fetching assets!";
```
Comment (Swizzy) [inline AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs:293]: This should probably be "Finished fetching assets..."

File: AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs [hunk 6]
```diff
@@ -319,7 +319,7 @@ public partial class OnlineAssetsControl {
                                          asset.GetAsset();
                                  };
                     bw.RunWorkerCompleted += (o, args) => {
-                                                 StatusMessage.Text = "Finished downloading asset data...";
+                                                 StatusMessage.Text = "Finished fetching assets!";
```
Comment (Swizzy) [inline AuroraAssetEditor/Controls/OnlineAssetsControl.xaml.cs:322]: This should probably be "Finished fetching assets..."

File: AuroraAssetEditor/MainWindow.xaml [hunk 7]
```diff
@@ -20,27 +20,27 @@
...
                     <Separator />
                     <MenuItem Click="ExitOnClick" Header="Exit" />
                 </MenuItem>
                 <MenuItem x:Name="SettingsMenu" Header="Settings">
                     <MenuItem x:Name="AutoResizeImages"
-                              Header="Auto Resize Images"
+                              Header="Automatically Resize Images"
                               IsCheckable="True"
-                              IsChecked="True" />
+                              IsChecked="False" />
```
Comment (Swizzy) [inline AuroraAssetEditor/MainWindow.xaml:41]: Why was this changed to false by default? the default behavior should be to automatically resize images to fit imho.

File: AuroraAssetEditor/Classes/XboxAssetDownloader.cs (hunk 1 above)
Comment (MrMendelli) [reply AuroraAssetEditor/Classes/XboxAssetDownloader.cs:180]: Fair enough, I can change this as well as the others you suggested.

File: AuroraAssetEditor/Classes/XboxAssetDownloader.cs (hunk 2 above)
Comment (MrMendelli) [reply AuroraAssetEditor/Classes/XboxAssetDownloader.cs:270]: Fair enough, I can change this as well as the others you suggested.

File: AuroraAssetEditor/MainWindow.xaml (hunk 7 above)
Comment (MrMendelli) [reply AuroraAssetEditor/MainWindow.xaml:41]: I changed this because I believe it will discourage simply stretching and warping the images. I should have specified this earlier, but I believe that if the end-user sees the warping they'll be encouraged to properly scale their images.

Comment (Swizzy) [reply AuroraAssetEditor/MainWindow.xaml:41]: Most users will not make their own covers, they'll be finding them online and putting in, therefor to cater to the majority of the user base, i would rather have this returned to the original default

Once you've updated with the adjustments mentioned, your pull request will be accepted/merged

Comment (MrMendelli) [reply AuroraAssetEditor/MainWindow.xaml:41]: Noted, I will push my commit tonight.

**Comment by Swizzy** [comment]
I assume you closed the pull request since it cannot be automatically merged due to the solution changes?

**Comment by MrMendelli** [comment]
> 
> 
> I assume you closed the pull request since it cannot be automatically merged due to the solution changes?

No, I screwed up my push/commit. The solution not merging automatically is due to me changing the source folder and project names _(`Source\Aurora Asset Editor.csproj`)_. I may just cancel this pull request all together, I changed these because of my own preferences but I feel like it will just get in the way of the project.
